docs/book/src/topics/machinepools.md

```diff
@@ -106,3 +103,34 @@ spec:
       kubeletExtraArgs:
         cloud-provider: aws
 ```
+
+### Example: Consume existing subnet IDs for AWSMachinePool
```

Comment:
subnets field is not specific to BYO VPC, removing this part all together and adding a subnet field above with subnet id is good enough.